### PR TITLE
Use attribute to request ovs installation

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

**Comment by rhafer** [comment]
This needs to be merged together with https://github.com/crowbar/crowbar-openstack/pull/80
For this reason the ci will not be able to succeed testing this PR.

**Comment by vuntz** [comment]
+1

**Comment by rhafer** [comment]
@vuntz I added a commit for cleaning up the openvswitch-switch vs openvswitch service issue on SLE12. Still ok with this?

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
LGTM

**Comment by rhafer** [comment]
Finally a real ci failure. I'll investigate.

**Comment by rhafer** [comment]
It failed because the crowbar-openstack packages is still outdated. I'll wait a little longer an retrigger the job once it's there.

File: chef/cookbooks/network/attributes/default.rb [hunk 1]
```diff
@@ -20,7 +20,12 @@
   default[:network][:ovs_pkgs] = ["openvswitch",
                                   "openvswitch-switch",
                                   "openvswitch-kmp-default"]
-  default[:network][:ovs_service] = "openvswitch-switch"
+  # SLES11 uses a different service name for openvswitch
+  if node[:platform] == "suse" && node[:platform_version].to_f < 12.0
```
Comment (toabctl) [inline chef/cookbooks/network/attributes/default.rb]: I think it needs to be ```:platform_family``` now!?

Comment (rhafer) [inline chef/cookbooks/network/attributes/default.rb]: No actually it should really only match SLES older than 12

**Comment by toabctl** [comment]
+1

**Comment by dirkmueller** [comment]
LGTM

**Pull request merged**